org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/flow/InsideStatementWithFinallyBlockFlowContext.java

```diff
@@ -33,7 +33,7 @@ public InsideSubRoutineFlowContext(
 
 @Override
 public String individualToString() {
-	StringBuilder buffer = new StringBuilder("Inside SubRoutine flow context"); //$NON-NLS-1$
+	StringBuilder buffer = new StringBuilder("Inside StatementWithPostScript flow context"); //$NON-NLS-1$
```

Comment:
I can't find this term in your dictionary ;)

Reply:
> I can't find this term in your dictionary ;)

:) Will fix along in a subsequent push.